ups/obs_subaru.build

```diff
@@ -1,2 +0,0 @@
-@LSST BUILD@ &&
-build_lsst @PRODUCT@ @VERSION@ @REPOVERSION@
```

Comment:
Why is this file going away?

Reply:
There was a conversation on Slack last week where we all came to the conclusion that nothing uses these files and they are unused cruft in the system that confuse people.

Reply:
That is correct.